src/dmd/backend/cgobj.d

```diff
@@ -79,16 +79,25 @@ version (MARS)
     extern(C) char* itoa(int,char*,int);
     extern(C) char* getcwd(char*,size_t);
 
+// Keep this in sync with struct Loc in globals.d?
```

Comment:
oh i didn't know about this copy of it. that might explain my crash

Reply:
turns out it's not needed... see last push